tests/ui_tests/test_component_page.py

```diff
@@ -201,15 +212,85 @@ def test_save_custom_config_on_component_config_page(self, app_fs, create_cluste
             app_fs.driver, app_fs.adcm.url, cluster.id, service.id, component.id
         ).open()
 
-        config_row = component_config_page.config.get_all_config_rows()[0]
-        component_config_page.config.type_in_config_field(row=config_row, value=params["row_value_new"], clear=True)
+        config_rows = component_config_page.config.get_all_config_rows()
+        with allure.step("Change value in float type on component config page"):
+            component_config_page.config.type_in_field_with_few_inputs(
+                row=config_rows[0], values=["1.1111111111"], clear=True
```

Comment:
can you get a needed config row by name instead of index number?
index looks unstable and harder to understand

Reply:
yep, i tried to figure out how to do it, but locators have no marks or anything about their types